common/misc.go

```diff
@@ -313,6 +313,6 @@ func GenerateMD5(file io.Reader, fileSize, maxSize int64) (hash string, hashSize
 
 // Parses the metadata and returns the value of the specified field
 func GetMetadataStringVal(metadata json.RawMessage, field string) string {
-	value, _ := jsonparser.GetString(metadata, field)
-	return value
+	valueBytes, _, _, _ := jsonparser.Get(metadata, field)
```

Comment:
should we rename the function as well? Since it's not only limited to string value

Reply:
I think it's fine since it returns the string value of the field, regardless of original type. What do you think?